scripts/importer/importer.py

```diff
@@ -0,0 +1,149 @@
+                        help = "Set dataset size for this dataset")
+    parser.add_argument("--set_vcf_sampleset_size", action="store_true",
+                        help = "Set/update sampleset size to the value given in the VCF."
+                               "This is either the NS value, or the number of stated samples")
+    parser.add_argument("--sampleset_size", type=int, default=0,
+                        help = "Set sampleset size for this dataset")
+    parser.add_argument("--beacon_description", default="",
+                        help="Set beacon description of the dataset.")
+    parser.add_argument("--assembly_id", default="",
+                        help="Set reference assembly id (GRC notation, e.g. GRCh37)")
+
+    # omim file, since we can't download or version them
+    parser.add_argument("--omim_file", default=os.path.join(os.path.dirname(__file__),
+                                                            "downloaded_files",
+                                                            "omim_info.txt.gz"),
+                        help = "OMIM annotation file.")
+
+    # Raw data (coverage + variants) files
+    parser.add_argument("--coverage_file", nargs="*",
+                        help = "Coverage file(s) to import.")
+    parser.add_argument("--variant_file", nargs="*",
+                        help = "Variant file(s) to import.")
+
+    # Actions
+    parser.add_argument("--add_reference", action="store_true",
+                        help = "Insert new reference set.")
+    parser.add_argument("--add_raw_data", action="store_true",
+                        help = "Adds a Coverage and Variants to the database.")
+    parser.add_argument("--add_dbsnp", action="store_true",
+                        help = "Adds a new dbSNP version to the database.")
+    parser.add_argument("--move_studies", action="store_true",
+                        help = ("Moves studies and datasets from an old database"
+                                " to a new one."))
+    parser.add_argument("--dry_run", action="store_true",
+                        help = "Do not insert anything into the database")
+
+    # Logging and verbosity
+    parser.add_argument("--disable_progress", action="store_true",
+                        help="Do not show progress bars.")
+    parser.add_argument("-v", "--verbose", action = "count", default = 3,
+                        help="Increase output Verbosity.")
+    parser.add_argument("-q", "--quiet", action = "count", default = 0,
+                        help="Decrease output Verbosity.")
+
+    # Beacon-only variants
+    parser.add_argument("--beacon-only", action="store_true",
+                        help="Variants are intended only for Beacon, loosening the requirements")
+
+    args = parser.parse_args()
+
+    logging.basicConfig(format="%(asctime)s %(levelname)s: %(message)s", level = (5-args.verbose+args.quiet)*10, datefmt="%H:%M:%S")
+
+    if args.add_dbsnp:
```

Comment:
More general question perhaps. But if we don't infer the dbsnp things when there's none, do we _really_ need to import this at all, can we just skip that in the database?

Reply:
dbsnp removed